Yosemite/Yosemite/Stores/WordPressSiteStore.swift

```diff
@@ -47,4 +49,20 @@ private extension WordPressSiteStore {
             }
         }
     }
+
+    func fetchApplicationPasswordAuthorizationURL(for siteURL: String, completion: @escaping (Result<URL?, Error>) -> Void) {
+        Task { @MainActor in
+            do {
+                let wpSite = try await remote.fetchSiteInfo(for: siteURL)
+                if let path = wpSite.applicationPasswordAuthorizationURL,
+                   let url = URL(string: path) {
+                    completion(.success(url))
+                } else {
+                    completion(.success(nil))
```

Comment:
Nit: What do you think about throwing a specific error instead of returning `nil`?

Reply:
IMO it's easier to spot the case where application passwords are not enabled with a nil URL than to define a new error. It also matches the response of the root endpoint `/wp-json`.